### PR TITLE
Mark some array tests as slow

### PR DESCRIPTION
**Pull request by mrocklin**
These five tests take up about half of the test runtime.

These still get tested on travis.ci, just not when run locally without
the --runslow keyword

### CODE REVIEW

**Comment by jcrist** [comment]
Fine by me.

> These five tests take up about half of the test runtime.

Do you mean for array only? I've noticed that `dask/dataframe/tests/test_multi.py` takes a long time as well. In my experience the array tests are quick compared to dataframe.

**Comment by mrocklin** [comment]
Just for arrays.  My experience mirrors your own.  I typically just run `py.test` on whatever module I'm working on at the time.  I've been working more with arrays lately.

File: dask/array/tests/test_fft.py [hunk 1]
```diff
@@ -111,6 +111,7 @@ def test_wrap_bad_kind():
         fft_wrap(np.ones)
 
 
+@pytest.mark.slow
 @pytest.mark.parametrize("funcname", all_nd_funcnames)
 @pytest.mark.parametrize("dtype", ["float32", "float64"])
 def test_nd_ffts_axes(funcname, dtype):
```
Comment (jakirkham) [inline dask/array/tests/test_fft.py]: Maybe a better fix for this one is to drop a dimension. This is using 4 dims to test with, but 3 is probably just as instructive. Besides if it is going to end up being skipped anyways, better to come up with a leaner version that we can still run.

Comment (mrocklin) [inline dask/array/tests/test_fft.py]: This will still be run if you run with the `--runslow` flag.  We call `py.test` with this flag on the CI services.

**Comment by mrocklin** [comment]
Merging soon if there are no further comments

File: dask/array/tests/test_fft.py (hunk 1 above)
Comment (jakirkham) [inline dask/array/tests/test_fft.py]: Ok, personally I'd still not like to see this test marked as slow. Tried cutting down the dimensionality of the test data for this test and it shaved ~3s off the test time of the FFT test suite. Please see PR ( https://github.com/dask/dask/pull/2270 ), which does exactly this.

**Pull request merged**